ramls/providers.raml

```diff
@@ -0,0 +1,160 @@
+          enum: [ "packages" ]
+          description: Name of nested resource to include
+          default: "packages"
+          required: false
+      responses:
+        200:
+          body:
+            application/vnd.api+json:
+              description: Provider details from EPKB for a specific provider ID.
+              example: !include examples/providers/providers_providerId_get_200_response.json
+        404:
+          description: Not Found
+          body:
+            application/vnd.api+json:
+              example: !include examples/providers/providers_providerId_404_response.json
+    put:
+      description: |
+        This operation allows you to update provider proxy and token values for a specific provider ID.
+      headers:
+        Content-Type:
+          example: application/vnd.api+json
+      body:
+        application/vnd.api+json:
+          properties:
+            data:
+              description: Needed because of JSON API
+              type: object
+              required: true
+              properties:
+                type: string
+                attributes:
+                  description: Needed because of JSON API
+                  type: object
+                  required: true
+                  properties:
+                    providerToken:
+                      displayName: Provider token
+                      type: object
+                      description: |
+                        Tokens are variables in content URLs that identify the customer.
+                        The token is text within the URL that needs to be replaced with an institute-specific value.
+                      example: |
+                        {
+                          "value": "hellotoken"
+                        }
+                      required: false
+                    proxy:
+                      displayName: Proxy ID
+                      type: proxy
+                      required: false
+          example: !include examples/providers/providers_providerId_put_request.json
+      responses:
+        200:
+          body:
+            application/vnd.api+json:
+              description: The server has successfully fulfilled the PUT request.
+              example: !include examples/providers/providers_providerId_put_200_response.json
+        500:
+          body:
+            application/vnd.api+json:
+              description: Unexpected error.
+    /packages:
+      get:
+        description: |
+          This operation allows you to retrieve a list of packages from EPKB for a provider including customer context.
+        is: [indexable, filterable]
+        responses:
+          200:
+            description: OK
+            body:
+              application/vnd.api+json:
+                example: !include examples/providers/providers_providerId_packages_get_200_response.json
+          400:
+            description: Bad Request
+            body:
+              application/vnd.api+json:
```

Comment:
On all the ramls/*.raml -> I am not sure if this actually builds using RMB -> did you give it a shot? When we put this RAML together using 1.0, since it is a Ruby module, it does not go through RMB and this RAML's mere purpose was to generate API documentation as we have it here: https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco/eholdings.html
If this is building through RMB and generating POJO's as expected, then well and good, otherwise, we'll have to revisit this.